tools/update_apt_archive

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+# Copyright 2019 The Project Buendia Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License.  You may obtain a copy
+# of the License at: http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distrib-
+# uted under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, either express or implied.  See the License for
+# specific language governing permissions and limitations under the License.
+
+set -e
+
+if [ "$1" = "-h" -o -z "$1" -o -z "$2" -o -z "$3" ]; then
+    echo "Usage: $0 <repository-dir> <branch> <artifact-dir>"
+    echo
+    echo "Updates the apt archive in projectbuendia/builds with the latest"
+    echo "CircleCI build artifacts for a given branch of the"
+    echo "projectbuendia/buendia project."
+    echo
+    echo "<branch> can be either 'master' or 'dev'."
```

Comment:
Specifying the branch as an argument gives me the impression that the script uses the `<branch>` argument to determine which branch to use.  But it looks like that's entirely determined by `<artifact-dir>`; the `<branch>` argument is only translated immediately to `stable` or `unstable`.  So, for example, someone could invoke this script with the wrong `<artifact-dir>` and its output could misleadingly say it was fetching artifacts from `master` when the artifacts were actually generated on `dev` or vice versa.

If I'm understanding the script correctly, what it does is update the specified suite of a specified apt archive with the packages from a specified directory.  It doesn't actually know anything about Git branches or CircleCI artifacts.  So it seems more logical to me to specify the suite directly as an argument (`stable` or `unstable`).  I know this means that the translation from master/dev to stable/unstable has to happen somewhere else (probably `config.yml`) but it still seems better—what do you think?

Reply:
Updated as suggested -- the suite selection code now lives in `.circleci/config.yml`